src/app/admin/home-admin/home-admin.component.html

```diff
@@ -2,5 +2,9 @@
   <h4>Area Adiministrativa</h4>
   <hr>
 
-  <button class="btn btn-primary" [routerLink]="['/autores']" routerLinkActive="router-link-active" >Novo Autor</button>
+  <div class="col-3 mx-auto">
+    <button class="btn btn-primary btn-lg btn-block" [routerLink]="['/autores']" routerLinkActive="router-link-active" >Novo Autor</button>
```

Comment:
o que esse atributo ```routerLinkActive```?  Se for para a situação de ter links e um ou outro deve ser marcado como ativo, era bom implementar a condicional aqui.

Reply:
Sim. Na realidade não era para ter o routerLink nesse caso. No meu Home, eu estou criando apenas os botões para ir para a tela de cadastro, sem a necessidade dele estar agora. Vou remover do código.